openmetadata-ui/src/main/resources/ui/cypress/integration/Pages/myData.spec.js

```diff
@@ -0,0 +1,86 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { visitEntityTab } from '../../common/common';
+import {
+  FOLLOWING_TITLE,
+  MYDATA_SUMMERY_OPTIONS,
+  MY_DATA_TITLE,
+  RECENT_SEARCH_TITLE,
+  RECENT_VIEW_TITLE,
+} from '../../constants/constants';
+
+describe('MyData page should work', () => {
+  beforeEach(() => {
+    cy.goToHomePage();
+  });
+
+  const checkRecentlyViewElement = (index, tab) => {
+    visitEntityTab(tab);
+    cy.wait(100); // to prevent from flicky behaviour, if page is not loaded it will wait for 100ms else it will continue testing
```

Comment:
I think cypress will take care of page loading. Can you remove this wait time and check if the test succeeds?

Reply:
I have tried that part, but it's flicky as we have lots of API calls for explore page which causes the page to re-render and cypress fail to click on the element. but sometimes it's able to click so to prevent that behaviour we need to wait until page loads properly